es-design-system/pages/molecules/es-progress-circle.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div>
+        <p class="pb-200">
+            Extended from <b-link href="https://github.com/EnergySage/es-cdgm/blob/main/frontend/components/CircularProgress.vue">
+                <code>es-cdgm</code> codebase
+            </b-link>
+        </p>
+        <h2>
```

Comment:
nit: I wonder if we could simplify this page to something like:

![Screen Shot 2023-06-23 at 12 44 07 PM](https://github.com/EnergySage/es-ds/assets/114260/7339d195-74f6-4b39-a364-a66bd2872142)

While changing the height of the component can be useful, it still basically looks the same across heights. It seems like changing in value is something that's more useful to visualize here.